citesphere/src/main/webapp/WEB-INF/views/admin/apps/show.jsp

```diff
@@ -26,12 +38,16 @@ $(function() {
 <th>Client Id</th>
 <th>Name</th>
 <th>Description</th>
+<th/>
 </tr>
+<c:set var="count" value="0" />
 <c:forEach items="${clientList}" var="client">
-<tr> 
+<c:set var="count" value='${count+1}' />
+<tr id="row${count}"> 
 <td>${client.clientId}</td>
 <td>${client.name}</td>
 <td>${client.description}</td>
+<td><input type="button" value="Delete" onclick="removeRow('<c:url value="/admin/apps/${client.clientId}?page=${currentPage - 1}&${_csrf.parameterName}=${_csrf.token}" />','${count}')" /></td>
```

Comment:
let's ask the user if he is sure he wants to delete the app and remind them that this would mean whatever application is using the attached credentials won't be able to authenticate anymore.